libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBTableColumn.java

```diff
@@ -35,7 +35,7 @@
  * columnName.
  */
 @Data
-@EqualsAndHashCode(exclude = {"name", "warning", "schemaName", "tableName", "ordinalPosition"})
+@EqualsAndHashCode(exclude = {"name", "warning", "schemaName", "tableName", "ordinalPosition", "keyType"})
```

Comment:
Because DBTableColumnEditor uses `Objects.equals(oldColumn, newColumn)` when judging whether two columns are equal. I think `keyType` should not be included when judging whether two columns are equal.